### DISCUSSION TITLE
MongoDB CDC Connector 怎么同步整个database 呢？

### DISCUSSION THREAD

**Discussion by zhangxuehong** [apache/flink-cdc#1365] [Q&A] [answered]
在 MongoDB CDC Connector 的配置中的 collection 有说明：Name of the collection in the database to watch for changes. If not set then all collections will be captured.  这种 sql 的方式该怎么写呢？

**Comment by Jiabao-Sun** [comment, marked as answer]
从2.2.0版本起支持正则匹配多库多集合。
在2.2.0之前，指定database，collection留空。
